src/entity/DAL/entityRepository.ts

```diff
@@ -30,8 +30,8 @@ const createEntityRepository = (dataSource: DataSource) => {
       return entityEntity;
     },
 
-    async findManyEntities(entities: Entity[]): Promise<EntityDb[] | undefined> {
-      const entityEntities = await this.findBy({ entityId: In(entities.map((e) => e.entityId)) });
+    async findManyEntitiesByIds(entities: Entity[]): Promise<EntityDb[] | undefined> {
+      const entityEntities = await this.findBy({ entityId: In(entities.map((e) => e.entityId)), fileId: In(entities.map((e) => e.fileId)) });
```

Comment:
add a comment to clarify it works because the ids are uuid's and therefore unique

Reply:
done.